### DISCUSSION TITLE
User Privacy

### DISCUSSION THREAD

**Discussion by itspasindu** [lllyasviel/Fooocus#2405] [Q&A] [answered]
How fooocus ai collected our generated images and other datas. Is safe for our privacy.

**Comment by mashb1t** [comment, marked as answer]
Fooocus does not collect any images or data.

**Reply by itspasindu** [reply]
Are the images generated through google colab safe?

**Reply by mashb1t** [reply]
what do you mean by safe? Please specify the exact question, thanks!

**Reply by itspasindu** [reply]
Can anyone access to our generated images

**Reply by mashb1t** [reply]
To access Colab a public URL is used (*.gradio.live), so in general everyone with this URL can access the generated images through the history log.
Add `--disable-image-log` if you fear somebody could access your link or when sharing it with other people for improved privacy.
